src/main/webapp/CreateProfilePage.html

```diff
@@ -8,22 +8,24 @@
     </head>
   <body >
     <div class="container">
-    <h1>Create Your Profile Page</h1>
+    <h1 style="text-align:center">Create Your Profile Page</h1>
       <form action="/profile-data" method="POST">
-      <div class="form-group col-md-2">
+      <div class="form-group col-md-5" style="margin: 0 auto;">
         <label for="name">Name:</label><br />
         <input type="text" class="form-control" name="name" maxlength="160"/><br />
       </div>
       <!--TODO(oyins): validate usernames to make sure there is a string-->
-      <div class="form-group col-md-2">
+      <div class="form-group col-md-5" style="margin: 0 auto;">
```

Comment:
I don't really think you need to use ```style="margin``` if you use the Boostrap margin classes correctly. Take a look at: https://getbootstrap.com/docs/4.0/utilities/spacing/